index.html

```diff
@@ -410,6 +463,94 @@ <h3 id="installation-sec">
       </h2>
       <section>
         <h3>
+          <code>BeforeInstallPromptEvent</code>
+        </h3>
+        <pre class="idl">
+enum AppBannerPromptOutcome {
+  "accepted",
+  "dismissed"
+};
+
+[Constructor(DOMString typeArg, optional BeforeInstallPromptEventInit eventInit)]
+interface BeforeInstallPromptEvent : Event {
+    readonly attribute Promise&lt;AppBannerPromptOutcome&gt; userChoice;
```

Comment:
>  I wonder if it's too late to change it?

I certainly wouldn't be opposed. I guess we need some telemetry data on usage to change it - and some coordination with existing sites that are using this. 

> I find the fact that this is a promise attribute (as opposed to a function that returns a promise) really weird. I know it's what Chrome does. I wonder if it's too late to change it?

There are lots of these in the platform, FWIW. https://github.com/heycam/webidl/issues/186#issuecomment-253526030